contrib/pg_get_functiondef/Makefile

```diff
@@ -0,0 +1,26 @@
+# contrib/pg_get_functiondef/Makefile
+
+MODULE_big = pg_get_functiondef
+OBJS = \
+	$(WIN32RES) \
+	pg_get_functiondef_no_input.o \
+	pg_get_functiondef_mul.o \
+	pg_get_functiondef_extend.o
+
+EXTENSION = pg_get_functiondef
+DATA = pg_get_functiondef--1.0.sql
+PGFILEDESC = "pg_get_functiondef - Enhancement of pg_get-functionaldef"
```

Comment:
Typo: pg_get-functionaldef should be pg_get_functiondef
Consider to reword PGFILEDESC to be like this: "Enhancement of existing pg_get_functiondef"